test/k8sT/Services.go

```diff
@@ -957,6 +1064,10 @@ var _ = Describe("K8sServicesTest", func() {
 						testExternalTrafficPolicyLocal()
 					})
 
+					It("Tests NodePort with sessionAffinity", func() {
+						testSessionAffinity()
+					})
+
 					It("Tests HealthCheckNodePort", func() {
 						testHealthCheckNodePort()
 					})
```

Comment:
Could you also enable the tests when XDP is enabled to make sure everything compiles and loads fine also from XDP side (since it's affecting it as well).

Reply:
It should be enabled for the XDP tests due to `kubeProxyReplacement=strict`.